operator/src/test/resources/runDeployedTests.sh

```diff
@@ -0,0 +1,15 @@
+cd ../../..
```

Comment:
Can you add a `shebang`?
I would move this bash script to the root folder of the operator since we don't need it to be included in the final `jar`

Reply:
> Can you add a `shebang`? I would move this bash script to the root folder of the operator since we don't need it to be included in the final `jar`

Good point.
We can do the same then with example-keycloak.yml on the java resources folder too.

Reply:
> We can do the same then with example-keycloak.yml

I agree, we can have an `example` folder in the root for it.